app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/registration/UserLocation.scala

```diff
@@ -22,20 +22,24 @@ import uk.gov.hmrc.eoricommoncomponent.frontend.services.cache.RequestSessionDat
 //TODO Make these sealed trait / case object with a isROW fn
 object UserLocation {
 
-  val Uk           = "uk"
-  val Iom          = "iom"
-  val Eu           = "eu"
-  val ThirdCountry = "third-country"
-  val Islands      = "islands"
+  val Uk                = "uk"
+  val Iom               = "iom"
+  val Eu                = "eu"
+  val ThirdCountry      = "third-country"
+  val ThirdCountryIncEU = "third-country-inc-eu"
+  val Islands           = "islands"
 
-  val validLocations: Set[String] = Set(Uk, Iom, Eu, ThirdCountry, Islands)
+  val validLocations: Set[String] = Set(Uk, Iom, Eu, ThirdCountry, ThirdCountryIncEU, Islands)
 
   def forId(locationId: String): Boolean = validLocations(locationId)
 
   def isRow(requestSessionData: RequestSessionData)(implicit request: Request[AnyContent]) =
     requestSessionData.selectedUserLocation match {
-      case Some(UserLocation.Eu) | Some(UserLocation.ThirdCountry) | Some(UserLocation.Islands) => true
-      case _                                                                                    => false
+      case Some(UserLocation.Eu) | Some(UserLocation.ThirdCountry) | Some(UserLocation.Islands) | Some(
+            UserLocation.ThirdCountryIncEU
+          ) =>
+        true
+      case _ => false
```

Comment:
To make it shorter we can create a list of RoW countries

`val rowCountries = List(UserLocation.Eu, UserLocation.ThirdCountry, UserLocation.Islands, UserLocation.ThirdCountryIncEU)` and change this method to

`requestSessionData.selectedUserLocation.map(rowCountries.contains(_)).getOrElse(false)`

but the current implementation is correct, just suggestion

Reply:
Yes.  That list is used twice isn't it.  Good idea.

Reply:
I was thinking of somewhere else where this list is used.  I'll try to find it.
But I've just realised that there are two methods for "isRow" in this class that will give different results if the user is from Iom.   More investigation required...

Reply:
Refactored like this as 'requestSessionData' is not properly mocked in many tests.   But it preserves the aim of the comment I think about grouping the "row" locations and I've re-used this logic in SubscriptionFlowManager.

The other isRow method was only used for Registration journey which would have excluded Iom anyway.  Hence the refactoring there.